docs/src/content/docs/hi/guides/sidebar.mdx

```diff
@@ -0,0 +1,541 @@
+---
+title: साइडबार मार्ग-निर्देशन
+description: जानें कि अपनी Starlight साइट के साइडबार मार्ग-निर्देशन लिंक को कैसे सेट अप और अनुकूलित करें।
+---
+
+import FileTree from '~/components/file-tree.astro';
+import SidebarPreview from '~/components/sidebar-preview.astro';
+
+एक सुव्यवस्थित साइडबार एक अच्छे दस्तावेज़ीकरण की कुंजी है क्योंकि यह उन मुख्य तरीकों में से एक है जिनसे उपयोगकर्ता आपकी साइट पर मार्ग-निर्देशन करेंगे। Starlight आपके साइडबार लेआउट और कंटेंट को अनुकूलित करने के लिए विकल्पों का एक पूरा सेट प्रदान करता है।
+
+## डिफ़ॉल्ट साइडबार
+
+डिफ़ॉल्ट रूप से, Starlight स्वचालित रूप से आपके दस्तावेज़ की संचिका प्रणाली संरचना के आधार पर साइडबार प्रविष्टि के रूप में प्रत्येक फ़ाइल की `title` गुण का उपयोग करके एक साइडबार उत्पन्न करेगा।
+
+उदाहरण के लिए, निम्नलिखित फ़ाइल संरचना दी गई है:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+      - reference/
+        - configuration.md
+
+</FileTree>
+
+निम्नलिखित साइडबार स्वचालित रूप से उत्पन्न हो जाएगा:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'guides',
+			items: [
+				{ label: 'अवयवों', link: '/guides/components/' },
+				{ label: 'अंतर्राष्ट्रीयकरण (i18n)', link: '/guides/i18n/' },
+			],
+		},
+		{
+			label: 'reference',
+			items: [
+				{ label: 'कॉन्फ़िगरेशन संदर्भ', link: '/reference/configuration/' },
+			],
+		},
+	]}
+/>
+
+[स्वतः-निर्मित समूह](#स्वतः-निर्मित-समूह) अनुभाग में स्वतः-निर्मित साइडबार के बारे में और जानें।
+
+## लिंक और लिंक समूह जोड़ें
+
+अपने साइडबार [लिंक](#लिंक) और [लिंक के समूहों](#समूह) (एक सिमटने वाले शीर्ष लेख के भीतर) को कॉन्फ़िगर करने के लिए, `astro.config.mjs` में [`starlight.sidebar`](/hi/reference/configuration/#sidebar) गुण का उपयोग करें।
+
+लिंक और समूहों को मिलाकर, आप विभिन्न प्रकार के साइडबार लेआउट बना सकते हैं।
+
+### लिंक
+
+`label` और `link` गुणों वाले ऑब्जेक्ट का उपयोग करके किसी आंतरिक या बाहरी पृष्ठ पर एक लिंक जोड़ें।
+
+```js "label:" "link:"
+starlight({
+	sidebar: [
+		// CSS और स्टाइलिंग मार्गदर्शिका का लिंक।
+		{ label: 'CSS और स्टाइलिंग', link: '/guides/css-and-tailwind/' },
```

Comment:
```suggestion
		{ label: 'CSS और स्टाइलिंग', link: '/hi/guides/css-and-tailwind/' },
```

We usually point to the proper version in this example so if the user clicks it by curiosity, it does not change its language. e.g. this [example](https://starlight.astro.build/pt-br/guides/sidebar/#links).